### PR TITLE
Allow override the compileSourceRoots. Fixes issue #53

### PR DESCRIPTION
**Pull request by mrmx**

### CODE REVIEW

**Comment by lgrignon** [comment]
Hello @mrmx 

Thanks a lot for contributing.

Did you test that normal case still works :) (not specifying the option)?
Could you please add a small description of the option in README.md file?

**Comment by mrmx** [comment]
Hi!

Yep added improvement to check emptiness and desc/example to the README too ;-)

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Enable the JSweet transpiler plugin for the preferred phase (here, generate-sour
 <plugin>
 	<groupId>org.jsweet</groupId>
 	<artifactId>jsweet-maven-plugin</artifactId>
-	<version>1.2.0-SNAPSHOT</version>
+	<version>2.4.0-SNAPSHOT</version>
```
Comment (lgrignon) [inline README.md:51]: thanks :)

**Comment by lgrignon** [comment]
Thanks @mrmx , that's nice

**Pull request merged**